bin/install.sh

```diff
@@ -349,6 +349,16 @@ echo WEB_CONCURRENCY=$webConcurrency >> $envFile
 
 echo -e "\n\nGreat! That's all we need, we will have your Waffle.io Takeout ready shortly."
 
+# POXA ENVs
+poxaAppId=`date +%s | sha256sum | base64 | head -c 16 ;`
+echo POXA_APP_ID=$poxaAppId >> $envFile
+
+poxaAppKey=`date +%s | sha256sum | base64 | head -c 32 ;`
+echo POXA_APP_KEY=$poxaAppKey >> $envFile
+
+poxaSecret=`date +%s | sha256sum | base64 | head -c 64 ;`
+echo POXA_SECRET=$poxaSecret >> $envFile
```

Comment:
If you run the install script multiple times do you end up with multiple entries for these?